starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -1,24 +1,30 @@
 from functools import partial
+from typing import Union, Tuple
 
 import numpy
 from skimage.filters import gaussian
 
-from starfish.munge import swap
 from ._base import FilterAlgorithmBase
 
 
 class GaussianLowPass(FilterAlgorithmBase):
 
-    def __init__(self, sigma, **kwargs) -> None:
+    def __init__(self, sigma, is_volume: bool=False, verbose=False, **kwargs) -> None:
         """Multi-dimensional low-pass gaussian filter.
 
         Parameters
         ----------
-        sigma : float
+        sigma : Union[float, Tuple[float]]
             Standard deviation for Gaussian kernel.
+        is_volume : bool
+            If True, 3d (z, y, x) volumes will be passed to self.low_pass. Otherwise, 2d tiles will be passed
```

Comment:
very confused here as `low_pass` does not seem to accept an is_volume parameter.

Reply:
Ah, so apply accepts it (and I have an issue out to expand this across all functions, #231). 

I can try to clarify the documentation here. Basically it provides 3d support by determining whether we apply gaussian_low_pass over planes or volumes.